scripts/annotation.js

```diff
@@ -4,7 +4,7 @@
  Minified.
  */
 $.urlParam = function (a) {
-    var b = new RegExp("[?&]" + a + "=([^&#]*)").exec(window.location.href);
+    let b = new RegExp("[?&]" + a + "=([^&#]*)").exec(window.location.href);
```

Comment:
If you're using WebStorm and chose ES6 as the interpreter, it will get quite mad at you if you use var :) 
Let specifies a specific scope for the variable and ensures that it is retained only in that scope, so it's best to use it to keep everything contained properly.